compliance/tags/azure_rg_tags/azure_resource_group_tags.pt

```diff
@@ -0,0 +1,300 @@
+    auth $azure_auth
+    pagination $azure_pagination
+    host "management.azure.com"
+    path join(["/subscriptions/", $param_azure_sub, "/resources"])
+    query "api-version","2018-02-01"
+    header "User-Agent", "RS Policies"
+  end
+  result do
+    encoding "json"
+    collect jmes_path(response, "value[*]") do
+      field "id", jmes_path(col_item,"id")
+      field "region", jmes_path(col_item,"location")
+      field "name", jmes_path(col_item,"name")
+      field "tags", jmes_path(col_item,"tags")
+    end
+  end
+end
+
+datasource "ds_filtered_resources" do
+  run_script $js_filter_resources, $ds_azure_resources, $param_tag_key
+end
+
+script "js_filter_resources", type: "javascript" do
+  parameters "ds_azure_resources", "param_tag_key"
+  result "result"
+  code <<-EOS
+    var result = [];
+
+    // Iterate through all resources
+    _.each(ds_azure_resources, function(ar){
+      console.log('Resource Name: '+ar.name)
+      var id = ar.id ;
+      var rg_split = id.split('/') ;
+      var resource_group_name = rg_split[4] ;
+      var namespace = rg_split[6] ;
+      var resource_type = rg_split[7] ; 
+      console.log('Resource Group Name: '+resource_group_name)
+      console.log('Namespace: '+namespace)
+      console.log('Resource Type: '+resource_type)
+
+      var resource_tags = ar.tags ;
+      if (namespace.match(/(Classic)/g)) {
+        // Skip Azure Classic
+        console.log('Skipping Azure Classic resource. ID: '+id)
+      } else {
+        if (_.isEmpty(resource_tags)) {
+          result.push({
+            id: ar["id"],
+            region: ar["region"],
+            name: ar["name"],
+            resource_group: resource_group_name,
+            namespace: namespace,
+            resource_type: resource_type,
+            tag: 'none'
+          })
+        } else {
+          if (_.has(resource_tags, param_tag_key)) {
+            var rg_tag_value = resource_tags[param_tag_key] ;
+            console.log('TAG KEY FOUND')
+            console.log('Tag Value: '+rg_tag_value)
+            result.push({
+              id: ar["id"],
+              region: ar["region"],
+              name: ar["name"],
+              resource_group: resource_group_name,
+              namespace: namespace,
+              resource_type: resource_type,
+              tag: rg_tag_value
+            })
+          } else {
+            console.log('Tag Key Not Found')
+            result.push({
+              id: ar["id"],
+              region: ar["region"],
+              name: ar["name"],
+              resource_group: resource_group_name,
+              namespace: namespace,
+              resource_type: resource_type,
+              tag: 'N/A'
+            })
+          }
+        }
+      }
+    })
+
+EOS
+end
+
+policy "policy_azure_resource" do
+  validate_each $ds_filtered_resources do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure Resources out of compliance"
```

Comment:
If its not specifically targeting an RS account, suggest removing this information from the summary_template. Could use the subscription id param instead.